app/models/curation_concerns/group.rb

```diff
@@ -0,0 +1,23 @@
+module CurationConcerns
+  class Group
+    def initialize(name)
+      @name = name
+    end
+
+    attr_reader :name
+
+    def to_sipity_agent
+      sipity_agent || create_sipity_agent!
```

Comment:
I could have done that. I was just following the patter @jeremyf used for `User`s.